CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 ## Unreleased
 * Added the `--docker-image` argument to the **lint** command, which allows determining the docker image to run lint on. Possible options are: `'native:ga'`, `'native:maintenance'`, `'native:dev'`, `'all'`, a specific docker image (from Docker Hub) or, the default `'from-yml'`.
 * Fixed an issue in **prepare-content** command where large code lines were broken.
+* Removed usgae of pipenv from the **split** command.
```

Comment:
```suggestion
* Removed pipenv generation from the **split** command.
* Running `autopep8` and `isort` is now done using their python libraries, rather than system calls.
```